myhdl/_Cosimulation.py

```diff
@@ -131,13 +131,18 @@ def _get(self):
         e = buf.split()
         for i in range(1, len(e), 2):
             s, v = self._toSigDict[e[i]], e[i+1]
-            try:
-                next = int(v, 16)
-                if s._nrbits and s._min is not None and s._min < 0:
-                    if next >= (1 << (s._nrbits-1)):
-                        next |= (-1 << s._nrbits)
-            except ValueError:
-                next = intbv(0)
+            if v in 'zZ':
```

Comment:
Thanks for the explanation (in the previous commit).  I think this makes sense and seems to be a reasonable solution.